default/python/AI/buildings.py

```diff
@@ -39,12 +46,28 @@ def available(self, eid: EmpireId = THIS_EMPIRE) -> bool:
             empire_object = fo.getEmpire(eid)
         return empire_object.buildingTypeAvailable(self.value)
 
+    def get_opinions(self) -> Opinion:
+        return get_planet_opinion(self.value)
```

Comment:
Let's remove this method and change `get_planet_opinion` to accept enum and string(temporary).